icu4c/source/test/cintltst/cloctst.c

```diff
@@ -5941,7 +5897,6 @@ const char* const locale_to_langtag[][3] = {
     {"th_TH_TH",    "th-TH-x-lvariant-th", NULL},
     {"bogus",       "bogus",        "bogus"},
     {"foooobarrr",  "und",          NULL},
-    {"az_AZ_CYRL",  "az-Cyrl-AZ",   "az-Cyrl-AZ"},
```

Comment:
instead of delete this test, could you instead put in the new expectation value instead?

Reply:
done